tests/by-util/test_shuf.rs

```diff
@@ -103,6 +124,74 @@ fn test_head_count() {
     );
 }
 
+#[test]
+fn test_head_count_multi_big_then_small() {
+    let repeat_limit = 5;
+    let input_seq = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+    let input = input_seq
+        .iter()
+        .map(ToString::to_string)
+        .collect::<Vec<String>>()
+        .join("\n");
+
+    let result = new_ucmd!()
+        .arg("-n")
+        .arg(&(repeat_limit + 1).to_string())
+        .arg("-n")
+        .arg(&repeat_limit.to_string())
+        .pipe_in(input.as_bytes())
+        .succeeds();
+    result.no_stderr();
+
+    let mut result_seq: Vec<i32> = result
+        .stdout_str()
+        .split('\n')
+        .filter(|x| !x.is_empty())
+        .map(|x| x.parse().unwrap())
+        .collect();
+    result_seq.sort_unstable();
```

Comment:
Why do you sort the result here and in `test_head_count_multi_small_then_big`?

Reply:
Good catch, thanks! That was a thoughtless copy-paste, because the other tests do it, too. Now that I think about it: You're right, it's not necessary, because the length of the result and membership of each (unique) expected element is already checked anyway.

Removed it in the next push.